### PR TITLE
90 sprint backend

### PR DESCRIPTION
**Pull request by oleksandr-vovk**
Just a backend part of sprint flow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +104,8 @@ erDiagram
         description string
         created_at datetime
         updated_at datetime
-        deadline datetime
+        start_at datetime
```
Comment (andpii) [inline README.md:107]: Does _start_at_ not the same as _created_at_? If no, then, please, rename _start_at_ to _started_at_ and _end_at_ to _finished_at_. And, i think, we should have property _deadline_ to know about theretical ending of the sprint.

File: backend/Tasque.Core.Common/Entities/Sprint.cs [hunk 2]
```diff
@@ -4,6 +4,17 @@ namespace Tasque.Core.Common.Entities;
...
 public class Sprint : BaseEntity
 {
+    public Sprint()
+    {
+        Tasks = new List<Task>();
+    }
+
     public string Name { get; set; } = null!;
     public string? Description { get; set; }
+    public DateTime? StartAt { get; set; }
```
Comment (andpii) [inline backend/Tasque.Core.Common/Entities/Sprint.cs:14]: Same

**Review by andpii — CHANGES_REQUESTED** [review]
~~And, please, update description of what you have added to database to our db scheme in README.~~ 
My bad, now i see it.

File: README.md (hunk 1 above)
Comment (oleksandr-vovk) [reply README.md:107]: created_at and updated_at is about working with entity. Those are service fields. But start_at and end_at are theoretical starting and ending dates.

Comment (andpii) [reply README.md:107]: I am talking about that _sprint_ can be finished after it's _deadline_. If you don't wanna rename them - that's ok, but don't remove _deadline_, please.

Comment (oleksandr-vovk) [reply README.md:107]: end _at is a deadline. We don't have fields for real ending date. That's why field names are in present but not in past tense.

**Review by andpii — APPROVED** [review]

**Review by ismv — APPROVED** [review]

**Pull request merged**